qupath-core/src/test/java/qupath/lib/plugins/parameters/TestParameterList.java

```diff
@@ -98,7 +98,7 @@ void testJson() {
 		assertNotEquals(json, jsonDifferent);
 
 		// Update the values, then check again
-		ParameterList.updateParameterList(paramsFromJson, mapFromJson, Locale.getDefault());
+		ParameterList.updateParameterList(paramsFromJson, mapFromJson, Locale.UK);
```

Comment:
QuPath's default is US - see https://github.com/search?q=repo%3Aqupath%2Fqupath%20locale.us&type=code

Reply:
```suggestion
		ParameterList.updateParameterList(paramsFromJson, mapFromJson, Locale.US);
```